From: John Hubbard <jhubbard@nvidia.com>
Subject: selftests/mm: .gitignore: add mkdirty, va_high_addr_switch
Date: Thu, 1 Jun 2023 18:33:52 -0700

These new build products were left out of .gitignore, so add them now.

Link: https://lkml.kernel.org/r/20230602013358.900637-7-jhubbard@nvidia.com
Signed-off-by: John Hubbard <jhubbard@nvidia.com>
Cc: Mel Gorman <mgorman@techsingularity.net>
Cc: Mike Kravetz <mike.kravetz@oracle.com>
Cc: Johannes Weiner <hannes@cmpxchg.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/testing/selftests/mm/.gitignore |    2 ++
 1 file changed, 2 insertions(+)

--- a/tools/testing/selftests/mm/.gitignore~selftests-mm-gitignore-add-mkdirty-va_high_addr_switch
+++ a/tools/testing/selftests/mm/.gitignore
@@ -39,3 +39,5 @@ local_config.h
 local_config.mk
 ksm_functional_tests
 mdwe_test
+mkdirty
+va_high_addr_switch
\ No newline at end of file
_
